crates/holochain/src/core/workflow/validation_receipt_workflow/unit_tests.rs

```diff
@@ -284,8 +284,13 @@ async fn errors_for_some_ops_does_not_prevent_the_workflow_proceeding() {
 
     assert_eq!(WorkComplete::Complete, work_complete);
 
-    // Should no longer require a receipt for either
-    assert!(!get_requires_receipt(vault.clone(), op_hash1).await);
+    // Sending the receipt to this author returned an error,
+    // so we did NOT clear the wants receipt flag.
+    assert!(get_requires_receipt(vault.clone(), op_hash1).await);
```

Comment:
This test was good to fix since the behavior changed. Now when we get an error the flag is not cleared, because we want to try sending the receipt again later.

And we still check that the second one that succeeded *does* get cleared.